modules/node_modules/@ncigdc/uikit/withToggle.js

```diff
@@ -1,20 +1,42 @@
 import React, { Component, PropTypes } from 'react';
+import _ from 'lodash';
 
 export default function withToggle(ComposedComponent) {
   return class Toggle extends Component {
+    static propTypes = {
+      // Allow active state changes to trigger actions such as toggling relay variables
+      onActivate: PropTypes.func,
+      onDeactivate: PropTypes.func,
+      active: PropTypes.bool,
+    };
+
+    static defaultProps = {
+      onActivate: _.noop,
```

Comment:
I've sometimes run into strangeness with the build tool attempting to call lodash2 when lodash4 was in package.json and then failing, especially when running jest tests. That's why I'd prefer to not use lodash if it's simple enough not to.